.ebextensions/cronjob.config

```diff
@@ -79,6 +79,16 @@ files:
         content: |
             0 6 * * * root /bin/bash -l -c 'cd /var/app/current && RAILS_ENV=production bundle exec rake alerts:create >> log/jobs.log 2>&1'
 
+    #
+    # Generate chart config daily
+    #
+    "/etc/cron.d/daily-chart-config-generation":
+        mode: "000644"
+        owner: root
+        group: root
+        content: |
+            0 6 * * * root /bin/bash -l -c 'cd /var/app/current && RAILS_ENV=production bundle exec rake configuration:generate_for_schools >> log/jobs.log 2>&1'
```

Comment:
Would this be better run just before the alerts?

Reply:
Yep, I'll re-jig

Reply:
Yes, moved it before the alerts